pire/stub/stl.h

```diff
@@ -43,6 +43,8 @@
 #include <stdexcept>
 #include <iterator>
 #include <functional>
+#include <tuple>
```

Comment:
We probably don't need `<tuple>` here without providing aliases such as something like ytuple and Tie. The only usage of `<tuple>` is fully-qualified call of `std::tie` in `count.cpp`. I suggest moving this include statement there.

Reply:
Right. Moved.